docs/도메인모델.md

```
@@ -0,0 +1,156 @@
+## toduck 일정 도메인
+
+- 유저는 일정을 생성할 수 있다
+    - 일정에는 단일 일정, 기간 일정이 있다.
+    - 종일 여부가 체크 된다면 시간은 선택할 수 없다. 그리고 알람이 1일전이거나 선택하지 말아야한다.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)